lib/update/phase_migration_test.go

```diff
@@ -18,45 +18,103 @@ package update
 
 import (
 	"context"
+	"path/filepath"
 	"time"
 
+	"github.com/gravitational/gravity/lib/compare"
 	"github.com/gravitational/gravity/lib/constants"
 	"github.com/gravitational/gravity/lib/ops/opsservice"
 	"github.com/gravitational/gravity/lib/storage"
+	"github.com/gravitational/gravity/lib/storage/keyval"
+	"github.com/gravitational/gravity/lib/users"
 
+	teleservices "github.com/gravitational/teleport/lib/services"
 	"github.com/pborman/uuid"
 	"gopkg.in/check.v1"
 )
 
 type PhaseMigrationSuite struct {
-	backend storage.Backend
+	backend   storage.Backend
+	cluster   storage.Site
+	backupDir string
 }
 
 var _ = check.Suite(&PhaseMigrationSuite{})
 
 func (s *PhaseMigrationSuite) SetUpSuite(c *check.C) {
 	services := opsservice.SetupTestServices(c)
 	s.backend = services.Backend
-}
-
-func (s *PhaseMigrationSuite) TestMigrateLinks(c *check.C) {
 	cluster, err := s.backend.CreateSite(storage.Site{
 		AccountID: uuid.New(),
 		Domain:    "example.com",
 		Created:   time.Now(),
```

Comment:
Tests should avoid variables as much as possible - better to use some predefined time constant.